orientdb/src/main/scala/akka/stream/alpakka/orientdb/scaladsl/OrientDBFlow.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.orientdb.scaladsl
+
+import akka.NotUsed
+import akka.stream.alpakka.orientdb._
+import akka.stream.scaladsl.Flow
+import com.orientechnologies.orient.core.record.impl.ODocument
+
+object OrientDBFlow {
+
+  /**
+   * Scala API: creates a [[OrientDBFlowStage]] that accepts as ODocument
+   */
+  def apply(
```

Comment:
In my taste this should be `create` just as in the Java API. It makes it more natural in case we add other constructors.